georaster-layer-for-leaflet.js

```diff
@@ -442,6 +446,56 @@ const GeoRasterLayer = L.GridLayer.extend({
     }
   },
 
+  /**
+   * The callback used to determine the colour based on the values of each pixel
+   *
+   * @callback pixelValuesToColor
+   * @param {array | number} values - The pixel value
+   * @returns {string} - Any valid CSS color string
+   */
+
+  /**
+   * Redraws the active map tiles updating the pixel values using the supplied callback
+   * @param {pixelValuesToColor} pixelValuesToColor - Callback that handles getting the pixel color
+   */
+  updateColors(pixelValuesToColor) {
+    if (!pixelValuesToColor) {
+      console.error("Missing pixelValuesToColor function");
+      return this;
```

Comment:
would we be able to throw this error, too?  And update from `pixelValuesToColor` to `pixelValuesToColorFn`?
`throw new Error("Missing pixelValuesToColorFn function");`

Reply:
Yeah sure thing.